actions/src/main/kotlin/com/alfresco/content/actions/ActionUploadExtensionFiles.kt

```diff
@@ -0,0 +1,52 @@
+package com.alfresco.content.actions
+
+import android.content.Context
+import android.net.Uri
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.alfresco.content.data.Entry
+import com.alfresco.content.data.OfflineRepository
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+/**
+ * Marked as ActionUploadFiles for uploading the files
+ */
+data class ActionUploadExtensionFiles(
+    override var entry: Entry,
+    override val title: Int = R.string.action_upload_files_title
+) : ActionExtension {
+
+    private val repository = OfflineRepository()
+
+    override suspend fun execute(context: Context, list: List<Uri>): Entry {
+        if (list.count() > 0) {
```

Comment:
We should use isEmpty instead of count()

Reply:
Kindly change it, we already discussed it before as well.

Reply:
Done